src/core/choroplethFeature.js

```diff
@@ -0,0 +1,247 @@
+//////////////////////////////////////////////////////////////////////////////
+/**
+ * Create a new instance of class choroplethFeature
+ *
+ * @class
+ * @extends geo.feature
+ * @returns {geo.choroplethFeature}
+ *
+ */
+//////////////////////////////////////////////////////////////////////////////
+geo.choroplethFeature = function (arg) {
+  'use strict';
+  if (!(this instanceof geo.choroplethFeature)) {
+    return new geo.choroplethFeature(arg);
+  }
+  arg = arg || {};
+  geo.feature.call(this, arg);
+
+  ////////////////////////////////////////////////////////////////////////////
+  /**
+   * @private
+   */
+  ////////////////////////////////////////////////////////////////////////////
+  var m_this = this,
+      s_init = this._init,
+      s_data = this.data,
+      m_choropleth = $
+      .extend({},
+              {
+                minColor: 'black',
```

Comment:
Hey @aashish24, thanks for the feedback! 
(1) For the value method, it's input is a geoJSON feature, not the geoJS feature class. It defines the accessor used to pull a value from the input data, which as I have designed it will always be geoJSON. Is this correct? 
(2) I think minColor and maxColor are useless, we should just use the color range for all values. It was a holdover from the contour class that doesn't really serve a purpose here, so I'll go ahead and remove those values. I also haven't implemented opacity scaling, not sure if that's useful for choropleths, what do you think?